library/objective-c/EnvoyEngine.h

```diff
@@ -122,6 +133,7 @@ typedef NSDictionary<NSString *, NSArray<NSString *> *> EnvoyHeaders;
 @property (nonatomic, assign) UInt32 dnsRefreshSeconds;
 @property (nonatomic, assign) UInt32 dnsFailureRefreshSecondsBase;
 @property (nonatomic, assign) UInt32 dnsFailureRefreshSecondsMax;
+@property (nonatomic, strong) NSArray *httpFilters;
```

Comment:
Should these be `NSArray<EnvoyHTTPFilter *> *` types?

Reply:
Sure 👍